src/encoded/static/components/navigation/components/UserActionDropdownMenu.js

```diff
@@ -56,7 +80,8 @@ export const UserActionDropdownMenu = React.memo(function UserActionDropdownMenu
         );
     } else {
         acctBtn = (
-            <LoginController {..._.pick(props, 'session', 'href', 'updateUserInfo', 'overlaysContainer', 'schemas', 'windowWidth')}>
+            <LoginController {..._.pick(props, 'session', 'href', 'updateUserInfo', 'overlaysContainer', 'schemas', 'windowWidth')}
+            auth0Options={FFAuth0Options}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 space characters but found 12. (react/jsx-indent-props)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4484611)